src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -1063,15 +1073,19 @@ public LocalDate fromString(String string) {
       setWinningRulesDefaultValues();
       switch (getWinnerElectionModeChoice(choiceWinnerElectionMode)) {
         case STANDARD_SINGLE_WINNER -> {
-          textFieldMaxRankingsAllowed.setDisable(false);
+          checkBoxMaxRankingsAllowedMax.setDisable(false);
+          checkBoxMaxRankingsAllowedMax
+              .setSelected(ContestConfig.SUGGESTED_MAX_RANKINGS_ALLOWED_MAXIMUM);
```

Comment:
I think it would be good practice to factor out all of the parts (like these two lines that you've added) that apply to all of the cases instead of repeating them across each case. You could just put them before the switch block.

Reply:
Ahh yeah, good point, I've been wanting to do this for a while but overlooked the fact that the switch wasn't driving the whole `onAction` thing. Done.